src/edu/asu/plp/tool/prototype/Main.java

```diff
@@ -413,45 +463,843 @@ private ProjectExplorerTree createProjectTree()
+		});
+		MenuItem itemOnlineManual = new MenuItem("Online Manual");
+		itemOnlineManual.setOnAction((event) -> {
+			onlineManualWeb();
+		});
+		MenuItem itemReportIssue = new MenuItem("Report Issue (Requires Google Account");
+		itemReportIssue.setOnAction((event) -> {
+			// TODO: Add Event for menu item
+		});
+		MenuItem itemGoogleIssues = new MenuItem("Open Google Code Issues Page");
+		itemGoogleIssues.setOnAction((event) -> {
+			// TODO: Add Event for menu item
+		});
+		MenuItem itemAboutPLP = new MenuItem("About PLP Tool...");
+		itemAboutPLP.setOnAction((event) -> {
+			// TODO: Add Event for menu item
+		});
+		MenuItem itemSWLicense = new MenuItem("Third Party Software License");
+		itemSWLicense.setOnAction((event) -> {
+			// TODO: Add Event for menu item
+		});
+		help.getItems().addAll(itemQuickRef, itemOnlineManual, new SeparatorMenuItem(),
+				itemReportIssue, itemGoogleIssues, new SeparatorMenuItem(), itemAboutPLP,
+				itemSWLicense);
+				
+		menuBar.getMenus().addAll(file, edit, view, project, tools, simulation, help);
+		
+		return Components.wrap(menuBar);
+	}
+	
 	private void onOpenProjectClicked(MouseEvent event)
 	{
 		console.println("Open Project Clicked");
 		openProjectFromFile();
 	}
+	
+	private void onSaveProjectClicked(MouseEvent event)
+	{
+		console.println("Save Project Button Clicked");
+		saveProjectFile();
+	}
+	
+	private void onAssembleProjectClicked(MouseEvent event)
+	{
+		console.println("Assemble Button Clicked");
+		Project activeProject = getActiveProject();
+		Optional<ISAModule> optionalISA = activeProject.getISA();
+		if (optionalISA.isPresent())
+		{
+			ISAModule isa = optionalISA.get();
+			Assembler assembler = isa.getAssembler();
+			// TODO: finish implementation
+			// assembler.assemble(activeProject);
+			throw new UnsupportedOperationException("Not yet implemented");
+		}
+		else
+		{
+			// TODO: handle "no compatible ISA" case
+			throw new UnsupportedOperationException("Not yet implemented");
+		}
+	}
+	
+	private Project getActiveProject()
+	{
+		Tab selectedTab = openProjectsPanel.getSelectionModel().getSelectedItem();
+		ASMFile activeFile = openProjects.getKey(selectedTab);
+		// TODO: check activeFile for null-value
+		return activeFile.getProject();
+	}
+	
+	private void onSimProjectClicked(MouseEvent event, HBox toolbar)
+	{
+		DropShadow ds = new DropShadow();
+		if (!simMode)
+		{
+			for (int x = 9; x <= 23; x++)
+			{
+				toolbar.getChildren().get(x).setEffect(null);
+				toolbar.getChildren().get(x).setDisable(false);
+			}
+			simMode = true;
+		}
+		else
+		{
+			for (int x = 9; x <= 23; x++)
+			{
+				toolbar.getChildren().get(x).setEffect(ds);
+				toolbar.getChildren().get(x).setDisable(true);
+			}
+			simMode = false;
+		}
+	}
+	
+	private void onlineManualWeb()
+	{
+		try
+		{
+			if (Desktop.isDesktopSupported())
+			{
+				Desktop.getDesktop().browse(new URI(
+						"https://code.google.com/p/progressive-learning-platform/wiki/UserManual"));
```

Comment:
This isn't bad, but it may be easier to debug/maintain if the inner argument is instantiated on a separate line, like so:
```
URI webAddress = new URI("https://code.google.com/p/progressive-learning-platform/wiki/UserManual");
Desktop.getDesktop().browse(webAddress);
```